Editor/Data/Type/DefaultObservableListDrawer.cs

```diff
@@ -2,139 +2,30 @@
 {
     using UnityEngine;
     using UnityEditor;
-    using System;
-    using Zinnia.Utility;
     using Zinnia.Data.Collection.List;
 
     [CustomPropertyDrawer(typeof(ObservableList), true)]
     public class DefaultObservableListDrawer : PropertyDrawer
     {
-        protected const string collectionLabel = "Elements";
-        protected const string collectionElementLabel = "Element {0}";
-        protected const string collectionEmptyLabel = "List is Empty";
-        protected const string invalidElementTypeMessage = "Element of Type `{0}` is not supported. Click the reference field `{1}` above and edit the `Elements` collection directly on the source component.";
-        protected const float footerSpacing = 2f;
-        protected const float objectPickerPadding = 4f;
-        protected const int indentResetLevel = -2;
-        protected readonly GUIStyle buttonStyle = new GUIStyle("RL Footer");
-        protected readonly GUIContent objectPickerIcon = EditorGUIUtility.TrIconContent("In ObjectField");
-        protected readonly GUIContent buttonIconAdd = EditorGUIUtility.TrIconContent("Toolbar Plus", "Add element to list");
-        protected readonly GUIContent buttonIconRemove = EditorGUIUtility.TrIconContent("Toolbar Minus", "Remove last element from list");
-
+        protected const string propertyReference = "elements";
+        protected readonly GUIContent propertyReferenceLabel = new GUIContent("Elements");
         public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
         {
-            label.tooltip = EditorHelper.GetTooltipAttribute(fieldInfo)?.tooltip ?? string.Empty;
             EditorGUILayout.PropertyField(property, label, true);
-            string fieldLabel = label.text;
+
             object propertyObject = fieldInfo.GetValue(property.serializedObject.targetObject);
-            if (propertyObject == null)
+            if (propertyObject == null || propertyObject.Equals(null))
```

Comment:
Is this necessary to check for Unity's equality and thus checking whether it's destroyed or not?

Perhaps alternatively we can just cast propertyObject to UnityEngine.Object instead above and only need the first null check here then?